python/app/shotgun_formatter.py

```diff
@@ -235,10 +235,10 @@ def _sg_field_to_str(self, sg_type, sg_field, value, directive=None):
             if not self._generates_links(value["type"]) or directive == "nolink":
                 str_val = link_name
             else:
-                str_val = "<a href='sgtk:%s:%s' style='text-decoration: none; color: %s'>%s</a>" % (value["type"], 
-                                                                                                    value["id"],
-                                                                                                    self._app.style_constants["SG_HIGHLIGHT_COLOR"], 
-                                                                                                    link_name)
+                str_val = self._app.frameworks["tk-framework-qtwidgets"].get_hyperlink_html(
```

Comment:
could also use: `sgtk.tank.platform.get_framework("tk-framework-qtwidgets")` I think. maybe that's preferable to accessing directly?